superset/assets/src/dashboard/components/Header.jsx

```diff
@@ -171,10 +171,10 @@ class Header extends React.PureComponent {
   }
 
   toggleEditMode() {
-    this.props.setEditMode(!this.props.editMode);
     this.props.logEvent(LOG_ACTIONS_TOGGLE_EDIT_DASHBOARD, {
       editMode: !this.props.editMode,
```

Comment:
umm should `editMode` be snake-cased `edit_mode`?

Reply:
fixed. thank you  👀 !